src/app/pages/TopicsPage/TopicsPage.jsx

```diff
@@ -0,0 +1,9 @@
+// render whatever data we get from getinitialdata i.e hello world
+
+import React from 'react';
+
+const TopicsPage = ({ pageData }) => {
```

Comment:
I think we need to add basic prop validation or an ignore lint comment to stop this breaking latest

Reply:
> I think we need to add basic prop validation or an ignore lint comment to stop this breaking latest

If we are planning to adopt TypeScript for this layer *soon-ish* then we'll have actual types set. What would be *better* right now, adding prop types or just ignoring the lint in your op?

Reply:
We won't be adding TypeScript here so need to add prop validation